pkgs/by-name/vt/vtsls/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nodejs_20,
+  git,
+  pnpm_8,
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
Isn't this required for following to work?

```nix
   rev = "server-v${version}";
```

Reply:
Use `rev = "server-v${finalAttrs.version}";` instead ;)

Reply:
Ah, sweet. Thanks!